app/views/dashboard/_new_hosts_widget_host_list.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class="table table-striped table-fixed">
+  <thead>
+    <th width="30%"><%= _('Host') %></th>
+    <th><%= _('Operating system') %></th>
+    <th><%= _('Owner') %></th>
+    <th><%= _('Created') %></th>
+    <th><%= _('Installed') %></th>
+  </thead>
+  <tbody>
+    <% hosts.each do |host| %>
+      <tr>
+        <td class="ellipsis"><%= link_to host.name, host_path(host) %></td>
```

Comment:
Ellipsis on the dashboard seems broken but it is unrelated to this. Opened http://projects.theforeman.org/issues/17501 to handle that.